### PR TITLE
add ibm ci test

### PR DESCRIPTION
**Pull request by MarkFranceschini**

### CODE REVIEW

File: ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml [hunk 1]
```diff
@@ -23,10 +27,18 @@ tag_specification:
   name: "4.3"
   namespace: ocp
 images:
-- from: base
-  to: hypershift-toolkit
+  - from: base
```
Comment (droslean) [inline ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml]: You have a wrong indentation here.

Comment (MarkFranceschini) [reply ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml]: Updated should be good now

File: ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml [hunk 2]
```diff
@@ -26,7 +30,15 @@ images:
 - from: base
   to: hypershift-toolkit
 tests:
-- as: verify
-  commands: make verify
-  container:
-    from: src
+  - as: verify
```
Comment (droslean) [inline ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml]: I didn't notice it before, sorry. You have wrong indentation here as well.

Comment (MarkFranceschini) [reply ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml]: updated !

**Pull request merged**